components/imgbb/actions/upload-picture/upload-picture.mjs

```diff
@@ -0,0 +1,41 @@
+// legacy_hash_id: "a_K5i2KK"
+import { axios } from "@pipedream/platform";
+import { stringify } from "qs";
+
+export default {
+  key: "imgbb-upload-picture",
+  name: "Upload picture",
+  description: "",
```

Comment:
Should we not include the `description` key in generated actions when it's an empty string?

Reply:
The `description` key is now excluded in generated action when it's an empty string.